PocketKit/Sources/PocketKit/Article/Cells/ArticleComponentTextCell.swift

```diff
@@ -60,6 +60,8 @@ class ArticleComponentTextView: UITextView {
         isScrollEnabled = false
         delegate = self
 
+        linkTextAttributes = [.foregroundColor: UIColor(named: "textfieldURL")]
```

Comment:
nice! the color looks so much better in dark mode! should we consolidate this hard coded string in the `Palettes` file to consolidate the asset strings?

Reply:
Should be better now!